scheduler/scheduler_unit_test.go

```diff
@@ -356,8 +367,51 @@ func (suite *SchedulerTestSuite) TestSchdulerDriverStopOK() {
 
 	suite.False(driver.Stopped())
 	suite.Equal(mesos.Status_DRIVER_RUNNING, driver.Status())
+	driver.connected = true // pretend that we're already registered
```

Comment:
What I really wanted to ask was if there is a way to reach the same state using only public methods instead of short-cutting and mutating internal state.